docs/tutorials/Drifter_data_tutorial.ipynb

```diff
@@ -0,0 +1,224 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Example Drifter data handling \n",
+    "### Agulhas retroflection region\n",
+    "The Atlantic and Indian Ocean meet around South Africa, and this is one of the most dynamic and energetic regions in the world ocean. The Agulhas retroflection is a region where the Agulhas current retroflects back into the Indian Ocean. This region is known for its strong currents and eddies, and is a region where many drifters have been deployed.\n",
+    "\n",
+    "In this tutorial, we will explore virtual drifters deployed in the Agulhas Current, just off the city of Durban at 31S. The virtual drifters report their position every hour, and have a lifetime of 30 days. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from virtualship import Location, Spacetime\n",
+    "from virtualship.instruments.drifter import Drifter, simulate_drifters\n",
+    "from virtualship.expedition.input_data import InputData\n",
+    "from pathlib import Path\n",
+    "import numpy as np\n",
+    "import xarray as xr\n",
+    "import matplotlib.pyplot as plt\n",
+    "import trajan\n",
+    "from datetime import timedelta"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We have downloaded the data from Copernicus Marine Service, using `virtualship fetch` and the information in following yaml format:\n",
+    "```yaml\n",
+    "area_of_interest:\n",
+    "  spatial_range:\n",
+    "    minimum_longitude: 5\n",
+    "    maximum_longitude: 60\n",
+    "    minimum_latitude: -45\n",
+    "    maximum_latitude: -30\n",
+    "    minimum_depth: -30\n",
+    "    maximum_depth: 0\n",
+    "  time_range:\n",
+    "    start_time: 2023-07-01 00:00:00\n",
+    "    end_time: 2023-09-30 00:00:00\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We have stored this hydrodynamic data in an `expedition_directory`, from which we can launch the virtual drifters. But before we do this, we will first need to load the data into a Parcels `FieldSet`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "expedition_directory = Path(\"/Users/erik/Desktop/VSC_AgulhasDrifters\")\n",
+    "fieldset = InputData._load_default_fieldset(expedition_directory)"
```

Comment:
I saw that you provided the output in other tutorials but thought here it would be nice to show how to run. So, I wanted to do the full download here, but since `virtualshiv fetch` is not yet at the point where it can supersmoothly be run, I decided to be a bit vague here. Depending on how fast the development of #83 goes, we could wait for that and then be more explicit here?